src/NumericalAlgorithms/DiscontinuousGalerkin/LiftFlux.hpp

```diff
@@ -42,6 +42,14 @@ auto lift_flux(Variables<tmpl::list<FluxTags...>> flux,
                Scalar<DataVector> magnitude_of_face_normal) noexcept
     -> Variables<tmpl::list<db::remove_tag_prefix<FluxTags>...>> {
   auto lift_factor = std::move(get(magnitude_of_face_normal));
+  // The LGL weights are:
+  //   w_i = 2 / ((N + 1) * N * P_{N}(xi_i))
+  // and so at the end points (xi = +/- 1) we get:
+  //   w_0 = 2 / ((N + 1) * N)
+  //   w_N = 2 / ((N + 1) * N)
+  // The negative sign comes from bringing the boundary correction over to the
+  // RHS of the equal sign, while the magnitude of the normal vector above
```

Comment:
I suggest clarifying "the equal sign". I know what you mean, but someone who hasn't derived DG a few times might want an extra bit of clarification here.

Reply:
I agree, though am struggling to figure out a better way to say that. Do you have a suggestion? I could do something like have the following in parens `du/dt = - boundary_correction`.

Reply:
Yeah, I think that would be an improvement. Or even `du/dt = (volume terms) - (boundary correction)` to not omit the other important piece :P

Reply:
Pff, who needs volume terms :P